e2e/launchertester/test_cases.go

```diff
@@ -130,10 +130,10 @@ func testSystemdUnits(t *testing.T) { //nolint: thelper, this is a test
 	terminateDistro(t)
 
 	distroNameToFailedUnits := map[string][]string{
-		"Ubuntu-18.04":   {"user@0.service", "atd.service", "systemd-modules-load.service"},
-		"Ubuntu-20.04":   {"user@0.service", "atd.service", "ssh.service", "systemd-remount-fs.service", "multipathd.socket"},
-		"Ubuntu-22.04":   {"user@0.service", "systemd-sysusers.service"},
-		"Ubuntu":         {"user@0.service", "systemd-sysusers.service"},
+		"Ubuntu-18.04":   {"user@0.service", "atd.service"},
+		"Ubuntu-20.04":   {"user@0.service", "atd.service", "systemd-remount-fs.service"},
```

Comment:
#370 should have addressed the `systemd-remount-fs` service. We didn't publish 20.04 and 18.04 apps since those patches were implemented, so the fixes are only testable with sideload builds.